File: 0001 Code Chef/0010 CodeChef Round/implementation/0004 java/CodeChefRound.java
import java.util.Scanner;


public class CodeChefRound{
    public static void main(String[] args){
        Scanner console = new Scanner(System.in);
        String stringInput = console.nextLine();
        int day = Integer.parseInt(stringInput);
        if(day == 4){
            System.out.println("YES");
        }else{
            System.out.println("NO");
        }
    }
}

File: 0001 Code Chef/0010 CodeChef Round/implementation/0009 dart/code_chef_round.dart
import 'dart:io';

void main(){
  String? stringInput = stdin.readLineSync();
  int? day = int.tryParse(stringInput!);
  if(day != null){
    if(day == 4){
      print("YES");
    }else{
      print("NO");
    }
  }
}